lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -12,54 +13,81 @@
 file_model_gh_sep = os.path.join(output_dir, 'cls_gh.sav')
 
 
+def run_program(*args):
```

Comment:
can this `run_program` function be useful somewhere else so it does not get defined inside a test file?

Reply:
Not really. Running this executables like this should only happen in tests of the scripts.